sceneformux/ux/src/main/java/com/google/ar/sceneform/ux/RotationController.java

```diff
@@ -15,45 +15,62 @@
  */
 package com.google.ar.sceneform.ux;
 
+import androidx.annotation.Nullable;
+
 import com.google.ar.sceneform.math.Quaternion;
 import com.google.ar.sceneform.math.Vector3;
 
 /**
  * Manipulates the rotation of a {@link BaseTransformableNode} using a {@link
  * TwistGestureRecognizer}.
  */
-public class RotationController extends BaseTransformationController<TwistGesture> {
+public class RotationController extends BaseTransformationController<TwistGesture> implements MovementController {
 
-  // Rate that the node rotates in degrees per degree of twisting.
-  private float rotationRateDegrees = 2.5f;
+  public class Settings {
+    // Rate that the node rotates in degrees per degree of twisting.
+    float rotationRateDegrees = 2.5f;
```

Comment:
This shouldn't still be private?

Reply:
They are public parameters that you can modify. For instance, settings of ScaleController are set to in SceneformVirtualObjectScalabilityConfiguration